.github/workflows/release.yml

```diff
@@ -43,7 +43,32 @@ jobs:
           retention-days: 1
           path: |
             dist/envd_linux_amd64_v1/envd
-            dist/envd_darwin_amd64_v1/envd
+            dist/envd_darwin_amd64_v1/envd      
+          if-no-files-found: error
+      - name: Get version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - name: Prepare publish manifest
+        run: |
+          mkdir manifest
+          mv dist/envd_linux_arm64/envd manifest/envd_${{ steps.get_version.outputs.VERSION }}_Linux_arm64
+          mv dist/envd_linux_amd64_v1/envd manifest/envd_${{ steps.get_version.outputs.VERSION }}_Linux_x86_64
+          mv dist/envd_linux_386/envd manifest/envd_${{ steps.get_version.outputs.VERSION }}_Linux_i386
+          mv dist/envd_darwin_amd64_v1/envd manifest/envd_${{ steps.get_version.outputs.VERSION }}_Darwin_x86_64
+          mv dist/envd_darwin_arm64/envd manifest/envd_${{ steps.get_version.outputs.VERSION }}_Darwin_arm64
+          mv dist/envd-sshd_linux_amd64_v1/envd-sshd manifest/envd-sshd_${{ steps.get_version.outputs.VERSION }}_Linux_x86_64
+          mv dist/envd-sshd_linux_arm64/envd-sshd manifest/envd-sshd_${{ steps.get_version.outputs.VERSION }}_Linux_arm64
+          mv dist/envd-sshd_linux_386/envd-sshd manifest/envd-sshd_${{ steps.get_version.outputs.VERSION }}_Linux_i386
+          mv dist/envd-sshd_darwin_amd64_v1/envd-sshd manifest/envd-sshd_${{ steps.get_version.outputs.VERSION }}_Darwin_x86_64
+          mv dist/envd-sshd_darwin_arm64/envd-sshd manifest/envd-sshd_${{ steps.get_version.outputs.VERSION }}_Darwin_arm64
+          mv dist/checksums.txt manifest/checksums.txt
+      - name: Upload publish manifest
```

Comment:
Why is this needed? We can just use `dist/`

Reply:
The file names in `dist/` have no version information. I've added these `mv` to keep the file name format consistent with the current release page.